doc/Reactive.md

```diff
@@ -16,42 +70,285 @@ Each [`Message<T>`](https://github.com/unoplatform/uno.extensions/blob/main/src/
+}
+```
+
+### Async
+Creates a feed from an async method. The loaded data can be refreshed using a `Signal` trigger that will re-invoke the async method.
+
+```csharp
+private IWeatherService _weatherService;
+
+public IFeed<WeatherInfo> Weather => Feed.Async(async ct => await _weatherService.GetCurrentWeather(ct));
+```
+
+### AsyncEnumerable
+This adapts an `IAsyncEnumerable<T>` into a _feed_
+
+```csharp
+private IWeatherService _weatherService;
+
+public IFeed<WeatherInfo> Weather => Feed.AsyncEnumerable(() => GetWeather());
+
+private async IAsyncEnumerable<WeatherInfo> GetWeather([EnumeratorCancellation] CancellationToken ct = default)
+{
+	while (!ct.IsCancellationRequested)
+	{
+		yield return await _weatherService.GetCurrentWeather(ct);
+		await Task.Delay(TimeSpan.FromHours(1), ct);
+	}
+}
+```
+
+### Create
+This gives you the ability to create your own _feed_ by dealing directly with _messages_.
+
+> This is designed for advanced usage and should probably not be used directly in apps.
+
+```csharp
+public IFeed<WeatherInfo> Weather => Feed.Create(GetWeather);
+
+private async IAsyncEnumerable<Message<WeatherInfo>> GetWeather([EnumeratorCancellation] CancellationToken ct = default)
+{
+	var message = Message<WeatherInfo>.Initial;
+	var weather = Option<WeatherInfo>.Undefined();
+	var error = default(Exception);
+	while (!ct.IsCancellationRequested)
+	{
+		try
+		{
+			weather = await _weatherService.GetCurrentWeather(ct);
+			error = default;
+		}
+		catch (Exception ex)
+		{
+			error = ex;
+		}
+
+		yield return message = message.With().Data(weather).Error(error);
+		await Task.Delay(TimeSpan.FromHours(1), ct);
+	}
+}
+```
+
+## Operators: How to interact with a feed
+You can apply some operators directly on any _feed_.
+
+> Note : You can use the linq syntax with feeds:
+> ```csharp
+> public IFeed<string> Value => from value in _values
+> 	where value == 42
+> 	select value.ToString();
+> ```
+
+### Where
+Applies a predicate on the _data_.
+
+Be aware that unlike `IEnumerable`, `IObservable` and `IAsyncEnumerable`, if the predicate returns false, a message with a `None` _data_ will be published.
+
+```csharp
+public IFeed<WeatherAlert> Alert => Weather
+	.Where(weather => weather.Alert is not null)
+	.Select(weather => weather.Alert!);
+```
+
+### Select
+Synchronously projects each data from the source feed.
+
+```csharp
+public IFeed<WeatherAlert> Alert => Weather
+	.Where(weather => weather.Alert is not null)
+	.Select(weather => weather.Alert!);
+```
+
+### SelectAsync
+Asynchronously project each data from the source feed.
+
+```csharp
+public IFeed<string> AlertDetails => Alert
+	.SelectAsync(async (alert, ct) => await _weatherService.GetAlertDetails(alert, ct));
+```
+
+### GetAwaiter
+This allows you to use `await` on a _feed_, for instance when you want to capture the current value to use it in a command.
+```csharp
+public async ValueTask ShareAlert(CancellationToken ct)
+{
+	var alert = await Alert;
```

Comment:
It might be worth specifying that 'alert' is a 'WeatherAlert' - currently the read has to scroll back up and find Alert and then work out what the return value type is